common.go

```diff
@@ -32,8 +32,3 @@ func PathSegmenter(path string, start int) (segment string, next int) {
 var (
 	segmenter = '/'
 )
-
```

Comment:
This is used in segmenter_test while testing `path_trie.go`.

Reply:
for now we keep it like this. I will remove in later comments, as some linting was failing